src/utils/orderUtils/parseOrder.ts

```diff
@@ -23,13 +23,30 @@ export interface ParsedOrder extends Order {
   surplusPercentage: BigNumber
   executedFeeAmount: string | undefined
   executedSurplusFee: string | null
-  formattedPercentage: number
-  parsedCreationTime: Date
+  filledPercentDisplayed: number
   executedPrice: Price<Currency, Currency> | null
   activityId: string | undefined
   activityTitle: string
 }
 
+export interface ParsedOrder {
+  id: string
+  isCancelling: boolean | undefined
+  inputToken: Token
+  outputToken: Token
+  kind: OrderKind
+  sellAmount: string
+  buyAmount: string
+  feeAmount: string
+  class: OrderClass
+  status: OrderStatus
+  partiallyFillable: boolean
+  parsedCreationTime: Date
```

Comment:
Isn't it redundant to prefix `parsed`?
I would think anything here is specific only to this type. Besides, it's a Date type rather than string like the original.

Reply:
It's definitely redundant, thanks!